factory/glideFactoryMonitorAggregator.py

```diff
@@ -631,13 +543,15 @@ def aggregateLogSummary():
     for entry in monitorAggregatorConfig.entries:
         # load entry log summary file
         status_fname = os.path.join(
-            os.path.join(monitorAggregatorConfig.monitor_dir, "entry_" + entry),
+            monitorAggregatorConfig.monitor_dir,
+            "entry_" + entry,
```

Comment:
Would it be better to change `"entry_" + entry` to `f"entry_{entry}"`?

Reply:
done